forge/db/views/Project.js

```diff
@@ -253,3 +259,10 @@ module.exports = function (app) {
         userProjectList
     }
 }
+
+function hasDashboard2UI (settings) {
```

Comment:
With this approach, when do these settings get updated? Do we need a restart of Node-RED for this to be recognised if installed by the "Manage Palette" option in the Editor?

Reply:
It should not require a restart.

When NR installs, it sends an event that ultimately populates settings.palette.modules.  When the page(s) are loaded, they get fresh version of the settings (including newly added modules).

I will add `deploy:pr` tag so this can be tried out / definitive test (since I dont have time today to actually try that scenario out)

Reply:
Tested on pre-staging and it performs as expected (requires page navigation or refresh)

![chrome_AqQk0WEENq](https://github.com/FlowFuse/flowfuse/assets/44235289/59f3467c-6cf5-48d5-85d9-50cb7f5fb7cc)